workspaces/blackduck/plugins/blackduck-backend/config.d.ts

```diff
@@ -0,0 +1,29 @@
+export interface Config {
+  blackduck: {
```

Comment:
```suggestion
  blackduck?: {
```

Is this truly mandatory? As is if this is missing I believe it will cause the backend to fail to start.